.github/workflows/main.yml

```diff
@@ -22,6 +22,10 @@ jobs:
           - stable
           - nightly
         os: [ubuntu-latest, macos-latest, windows-latest]
+        feature: []
```

Comment:
> The workflow is not valid. .github/workflows/main.yml (Line: 25, Col: 18): Matrix vector 'feature' does not contain any values

The default could be "serde" so that we can simplify the below command on `check build`.

Reply:
nit: And it should be `features`.